etc/kayobe/kolla/config/nova.conf

```diff
@@ -0,0 +1,3 @@
+[libvirt]
+hw_machine_type = q35
+num_pcie_ports = 16
```

Comment:
That's going to be backported upstream (right @bbezak ?) - so we probably could set only hw_machine_type?

Reply:
master got merged today. backports just started - https://review.opendev.org/c/openstack/kolla-ansible/+/895373

Reply:
stable/yoga merged

Reply:
Probably we should have kept this since we can't assume kolla-ansible is up to date on client deployments.